packages/playground/src/components/contracts_list/contracts_table.vue

```diff
@@ -145,11 +177,68 @@
       </v-card-actions>
     </v-card>
   </v-dialog>
+
+  <v-dialog width="500" v-model="unlockDialog">
+    <v-card>
+      <v-card-title class="text-h5 pa-0">
+        <v-toolbar color="primary" dark class="custom-toolbar">
+          <p class="mb-9 mt-4">
+            Unlock the following Contract<span v-if="selectedContracts.length > 1">s</span>
+          </p></v-toolbar
+        >
+      </v-card-title>
+      <v-card-text>
+        <v-alert class="my-4" type="warning" variant="tonal">
+          <div>
+            Selected contract<span v-if="selectedContracts.length > 1">s</span> Locked amount:
+            <span class="font-weight-bold">{{ lockedAmount.toFixed(4) }} </span> TFTs
```

Comment:
We normally use `toFixed(3)` to wrap any TFT value, why here you used `toFixed(4)`?